fenix/app/src/test/java/org/mozilla/fenix/browser/TranslationsBindingTest.kt

```diff
@@ -149,6 +159,11 @@ class TranslationsBindingTest {
                     selectedTabId = tabId,
                 ),
             )
+            browserStore.dispatch(
```

Comment:
If you can think of a test for the browser menu entry point, that would be nice to have too.